Update peer dep

Hi, is it possible to update the peer dep of this project to react 17?

```npm ERR! peer dep missing: react@^16.8.0, required by react-google-places-autocomplete@3.0.0```

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "use-debounce": "^3.4.3"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react": "^16.8.0 || ^17.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0"
   }
 }

